test/srt/run_suite.py

```diff
@@ -9,23 +9,23 @@
         "models/test_generation_models.py",
         "models/test_lora.py",
         "models/test_reward_models.py",
-        "sampling/penaltylib",
         "test_chunked_prefill.py",
         "test_embedding_openai_server.py",
         "test_eval_accuracy_mini.py",
+        "test_srt_engine.py",
         "test_json_constrained.py",
         "test_large_max_new_tokens.py",
         "test_openai_server.py",
         "test_pytorch_sampling_backend.py",
         "test_server_args.py",
         "test_skip_tokenizer_init.py",
-        "test_srt_engine.py",
         "test_srt_endpoint.py",
         "test_torch_compile.py",
         "test_torchao.py",
         "test_triton_attn_backend.py",
         "test_update_weights.py",
         "test_vision_openai_server.py",
+        "sampling/penaltylib",
```

Comment:
why do you change the order here? They are sorted alphabetically.

Reply:
this is for my convenience to let CI run my test sooner. forgot to revert back. let me do it